README.md

```diff
@@ -1,93 +1,34 @@
 # Forum-Project-Stage-CC
```

Comment:
Why are you removing the Project Vision Statement from the Redme file?

Reply:
Please make a separate "local installation" section and let Project Vision be present.